cypress/integration/workflowDetails.js

```diff
@@ -1,13 +1,35 @@
+describe('Variations of URL', function() {
```

Comment:
I may refer to these two as a FAQ entry actually, might be worth telling people what we mean by "canonical URL."